### PR TITLE
feat: Bootstrap more aggresively when too few peers

### PR DESCRIPTION
**Pull request by zvolin**

### CODE REVIEW

File: node/src/p2p.rs [hunk 1]
```diff
@@ -426,13 +440,13 @@ where
...
-                    // scanning the network thought the already known peers and find new
-                    // ones. It also recovers connectivity of previously known peers and
-                    // refreshes the routing table.
-                    //
-                    // libp2p team suggests to start bootstrap procedure every 5 minute.
-                    let _ = self.swarm.behaviour_mut().kademlia.bootstrap();
+                    if self.peer_tracker.info().num_connected_peers < MIN_CONNECTED_PEERS
+                        || kademlia_last_bootstrap.elapsed() > KADEMLIA_BOOTSTRAP_PERIOD
+                    {
+                        info!("Running kademlia bootstrap procedure.");
```
Comment (oblique) [inline node/src/p2p.rs:446]: I think I would log this as debug, but up to you.

**Review by oblique — APPROVED** [review]
lgtm. minor comments

File: node/src/p2p.rs (hunk 1 above)
Comment (zvolin) [reply node/src/p2p.rs]: done

**Review by oblique — APPROVED** [review]

**Review by fl0rek — APPROVED** [review]
🎸

**Pull request merged**